ports/litex/boards/orangecrab/csr.h

```diff
@@ -0,0 +1,736 @@
+//--------------------------------------------------------------------------------
+// Auto-generated by Migen (12e7ba6) & LiteX (e7646416) on 2020-07-11 11:48:20
```

Comment:
Should this be under `generated` too?

Reply:
Yes, that would make sense.

Reply:
Should we also move the `csr.h` in fomu too?

In tinyusb the litex port references just `#include "csh.h"`. So would adding an `-I` flag to `CFLAGS` be a good option? Or should we be editing all the source files with `#include "generated/csh.h"`

Reply:
Yes please move the Fomu copy too. You can add the `-I` for TinyUSB but I prefer `generated/csr.h` in CircuitPython because it's clearer.